lib/irclogger/viewer_helpers.rb

```diff
@@ -5,11 +5,11 @@ module ViewerHelpers
     include Rack::Utils
 
     def channel_escape(channel)
-      channel[1..-1].gsub(/^#+/) { |m| '.' * m.length }
+      CGI.escape(channel[1..-1]).gsub(/^#+/) { |m| '.' * m.length }
```

Comment:
So... what does the slash become? Some %-encoded atrocity? How about replacing it with a `⁄`?